packages/web-components/src/components/cbp-toggle/cbp-toggle.stories.tsx

```diff
@@ -81,17 +79,16 @@ function generateToggles(items, labelWidth, hideStatus, statusTexton, statusText
         ${context && context != 'light-inverts' ? `context=${context}` : ''}
         ${sx ? `sx=${JSON.stringify(sx)}` : ``}
         >
-            <label
-                slot='toggleLabel'
+            <span
                 style='width: ${labelWidth}'
```

Comment:
I'm not a fan of this implementation. I would make it a prop on the component and use it to set a CSS variable in sx that gets used in the component CSS. The span should not be needed then.

Reply:
update as discussed in call